flink-runtime/src/main/java/org/apache/flink/runtime/security/SecurityUtils.java

```diff
@@ -230,10 +230,15 @@ private void validate() {
 		}
 
 		private static List<String> parseList(String value) {
-			if(value == null) {
+			if(value == null || value.isEmpty()) {
 				return Collections.emptyList();
 			}
-			return Arrays.asList(value.split(","));
+
+			return Arrays.asList(value
+				.replaceAll("\\s*,\\s*", ",") // remove whitespaces surrounding commas
+				.replaceAll(",,", ",") // remove empty entries
```

Comment:
Hi, @tzulitai `.replaceAll("\\s*,\\s*", ",").replaceAll(",,", ",") ` can not deal with `" a, b,,, c d, e "`
I think we can using the expression as follows:
 `str.trim().replaceAll("(\\s*,+\\s*)+", ",")` OR `str.replace("/^\\s+|\\s+$/g","").replaceAll("(\\s*,\\s*)+", ",");`